ecs.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+use Symplify\EasyCodingStandard\Config\ECSConfig;
+use Symplify\EasyCodingStandard\ValueObject\Set\SetList;
+
+return static function (ECSConfig $ecsConfig): void {
+    $ecsConfig->paths([__DIR__]);
+    $ecsConfig->skip([__DIR__ . '/vendor']);
+
+    $ecsConfig->sets([SetList::ARRAY]);
+};
```

Comment:
Using ECS instead of PHP-CS-Fixer as ECS has a lot better support for formatting arrays which is very critical here.